src/components/form/Steps/Step2.tsx

```diff
@@ -33,60 +21,91 @@ export default function Step2({ onClickBack }: Step2Props) {
       message: "Preencha o campo ${label}!",
     },
   ];
+  const rulesLatitude = [
+    {
+      required: false,
+      pattern: /^-?([1-8]?\d|90)(,\d{1,7})?$/,
+      message:
+        "Deve estar entre -90 e +90 e até 7 casas decimais, utilizando vírgula!",
+    },
+  ];
 
-  const [opcoesUf, setOpcoesUf] = useState<FederativeUnit[]>([]);
-  const getUf = async () => {
-    try {
-      const resposta = await fetchFederativeUnit();
-      setOpcoesUf(resposta);
-    } catch (error) {}
-  };
-  useEffect(() => {
-    if (opcoesUf.length == 0) getUf();
-  });
+  const rulesLongitude = [
+    {
+      required: false,
+      pattern: /^-?((1?[0-7]|[0-9])?\d|180)(,\d{1,7})?$/,
+      message:
+        "Deve estar entre -180 e +180 e até 7 casas decimais, utilizando vírgula!",
+    },
+  ];
+
+  const rulesCodigoEscola = [
+    {
+      required: true,
+      pattern: /^\d{8}$/,
+      message: "O código deve conter 8 digitos",
+    },
+  ];
 
-  const [opcoesMunicipio, setOpcoesMunicipio] = useState<Municipio[]>([]);
-  const getMunicipio = async () => {
+  const rulesTelefone = [
+    {
+      required: true,
+      pattern: /^\d{10,11}$/,
+      message: "O telefone deve conter DDD + 9 ou 8 digitos",
+    },
+  ];
+
+  const rulesNumeroAlunoDocentes = [
+    {
+      required: true,
+      pattern: /^[1-9]\d*$/,
+      message: "Deve conter apenas números",
+    },
+  ];
+
+  const getCEP = async (cep: string) => {
     try {
-      if (UFSelecionada) {
```

Comment:
Nessa função de GetCEP precisamos colocar uma condição para desabilitar os campos UF e municipio se o CEP for encontrado com sucesso, e caso não encontra, ai permite inserir os campos manualmente

Lembre-se de colocar uma mensagem também